src/main/java/cubicchunks/server/ChunkGc.java

```diff
@@ -73,4 +75,9 @@ private void chunkGc() {
             }
         }
     }
+
+    @Override
+    public void onConfigUpdate(Config config) {
+        this.updateInterval = config.getChunkGCInterval();
```

Comment:
Note that this is very likely to be called from other thread. Technically `updateInterval` should be volatile if you want to guarantee that changes made in this thread are visible to other threads.